ci/do_ci.sh

```diff
@@ -142,12 +142,15 @@ elif [[ "$CI_TARGET" == "bazel.debug.server_only" ]]; then
   exit 0
 elif [[ "$CI_TARGET" == "bazel.asan" ]]; then
   setup_clang_toolchain
+  BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} -c dbg --config=clang-asan \
```

Comment:
BAZEL_BUILD_OPTIONS+=" -c dbg ...

Reply:
qq: why can't these options be in the bazelrc so they work for local dev also?

Reply:
@mattklein123 because this adds options like `--linkopt=-L/opt/llvm/lib/clang/9.0.0/lib/linux` (from the bash vars), which is very specific to the environment and that will break for anywhere other than CI image. I can make it work by generating .bazelrc as follow up though.

Reply:
Cool sounds good regarding follow up. Can you add a comment about that?

Reply:
clarified in .bazelrc comment